scripts/paper/generate_proof_pressure_boundaries_figures.py

```
@@ -383,10 +390,86 @@ def render_mechanism_figure() -> None:
     write_figure(fig, "proof-pressure-opening-mechanism-2026-05")
 
 
+def render_d64_high_query_figure() -> None:
+    rows = read_tsv(D64_HIGH_QUERY_TSV)
+    if not rows:
+        raise SystemExit(f"D64_HIGH_QUERY_TSV is empty: {D64_HIGH_QUERY_TSV}")
```

Ruff (0.15.18)

[warning] Avoid specifying long messages outside the exception class

(TRY003)